### ISSUE TITLE
[2020] Day 2: Data Wrangling Fundamentals

### ISSUE THREAD

**Issue by gvegayon** [USCbiostats/rbootcamp#18]

**Comment by erickawaguchi** [comment]
I can probably help with #18.

**Comment by gvegayon** [comment]
You should be looking then at the materials in https://github.com/USCbiostats/rbootcamp/tree/fall2019/day1 and some other materials I sent you by email :)

**Comment by malcolmbarrett** [comment]
Oh it looks like last years materials for this aren't in the 2019 snapshot? I guess I didn't update them on this repo. They are actually here: https://github.com/malcolmbarrett/rbootcamp_data_analysis

**Comment by gvegayon** [comment]
Yeah, we need to update the structure... we can figure that out later :).
For now, I'm already working on this year's structure (will push in the
next few minutes). Now go back to your work :P!

George G. Vega Yon
+1 (626) 381 8171
https://ggvy.cl

**Comment by malcolmbarrett** [comment]
👀 👀 👀
